docs/hyperopt.md

```diff
@@ -198,7 +198,7 @@ Hyperopt will then spawn into different processes (number of processors, or `-j
 
 For every new set of parameters, freqtrade will run first `populate_entry_trend()` followed by `populate_exit_trend()`, and then run the regular backtesting process to simulate trades.
 
-After backtesting, the results are passed into the [loss function](#loss-functions), which will evaluate if this result was better or worse than previous results.  
+After backtesting, the results are passed into the [loss function](#loss-functions), which will evaluate if this result was better or worse than previous results.
```

Comment:
Unfortunately it's ... not just "tidying" - not in all instances, anyway. 
2 spaces have a special meaning in markdown, and mean "new line without new paragraph" - so this will now change (quite drastically) how it renders.

from 
![image](https://github.com/freqtrade/freqtrade/assets/5024695/1e41c3d5-fc20-412a-b7a5-0be42875c255)

to 
![image](https://github.com/freqtrade/freqtrade/assets/5024695/fabfa56b-38a8-4794-bb22-58c114006fac)

(all mashed together).

For most changes - it's fine (like in the code blocks, ...) - though in the (few) cases where the following regex matched - we need to revert this (regex `\s\s$` - basically 2 spaces followed by a newline)

---

i assume you're using something like the following setting - and i'd advise to add the markdown section (the below is vscode config ...).


```
    "files.trimTrailingWhitespace": true,
    "[markdown]": {
        "files.trimTrailingWhitespace": false,
    },
```